README.md

```diff
@@ -116,35 +124,19 @@ try {
         }
     });
 }
-catch(SinchApiException e) 
+catch(SinchApiException e)
 {
     logger.LogError("Api Exception. Status: {status}. Detailed message: {message}", e.Status, e.DetailedMessage);
 }
 ```
 
-## Logging, HttpClient, and additional options
+## Sample apps
 
-To configure logger, provide own `HttpClient`, and additional options utilize `SinchOptions` within constructor:
+For additional examples see [examples](https://github.com/sinch/sinch-sdk-dotnet/tree/main/examples)
```

Comment:
It might be useful to have a dedicated README in the examples directory to help new users familiarize themselves with the details. 
(About examples: not sure this one is related to one of our supported API: https://github.com/sinch/sinch-sdk-dotnet/blob/main/examples/WebApi/WeatherForecast.cs)

Reply:
This one too: https://github.com/sinch/sinch-sdk-dotnet/blob/main/examples/WebApi/Controllers/WeatherForecastController.cs

Reply:
@asein-sinch cleaned 👌